squad/frontend/templates/squad/build-nav.html

```diff
@@ -0,0 +1,20 @@
+{% load squad %}
+<h1> <a href="{% group_url project.group %}">{{project.group.slug}}</a>/<a href="{% project_url project %}">{{project.slug}}</a>/<a href="{% build_url build %}">{{build.version}}</a></h1>
+  <ul class="nav nav-tabs">
+    {% with url_name=request.resolver_match.url_name %}
+
+    <li role="presentation" {% if url_name == 'tests' or url_name == 'test_history' %}class="active"{% endif %}>
+      <a href="{% build_section_url build 'tests' %}">
+        Tests
+      </a>
+    </li>
+
+    <li role="presentation" {% if url_name == 'metrics' %}class="active"{% endif %}>
+      <a href="{% build_section_url build 'metrics' %}">
+        Metrics
+      </a>
+    </li>
```

Comment:
metrics are not per-build(that is the charts page with history data for each metric), so it seems a little weird to have them linked from the build page.